ui/imports/shared/views/chat/MessageContextMenuView.qml

```diff
@@ -29,7 +29,7 @@ StatusMenu {
     property string selectedUserDisplayName: ""
     property string selectedUserIcon: ""
 
-    property int chatType: Constants.chatType.publicChat
+    property int chatType: Constants.chatType.unknown
```

Comment:
Isn't it should be 'oneToOne'? `unknown` is not handled in that [switch](https://github.com/status-im/status-desktop/pull/9253/files#diff-54c7d5c23951abe7c4bd4df905b5fc2a58bbdd03ba320183e128eb8d7d17830aR413)